src/ophyd_async/core/sim_signal_backend.py

```diff
@@ -152,14 +152,18 @@ def _set_value(self, value: T):
         if self.callback:
             self.callback(reading, self._value)
 
+    def _get_value(self):
+        """Method to bypass asynchronous logic, designed to only be used in tests."""
+        return self.converter.value(self._value)
+
```

Comment:
Is this used anywhere?

Reply:
nevermind; turns it out is used!

Reply:
Where is it used apart from `self.get_value()`?

Reply:
aha. I've now removed it... must have had a brain fart moment, as I remember removing it and seeing tests fail, and not investigating to discover that self.get_value is the only place this is called. Whoops. Oh well, it's removed now.